nautobot_circuit_maintenance/handle_notifications/sources.py

```diff
@@ -407,10 +420,14 @@ class GmailAPI(EmailSource):
     service: Optional[Resource] = None
     credentials: Optional[Union[service_account.Credentials, Credentials]] = None
 
-    SCOPES = ["https://www.googleapis.com/auth/gmail.readonly"]
+    SCOPES = [
+        "https://www.googleapis.com/auth/gmail.readonly",
+        "https://www.googleapis.com/auth/gmail.modify",
```

Comment:
what if someone has concerns on autorizing this new scope and doesn't care about this feature? I believe we should leave this open and maybe add it as an extra_scope if explicitly defined via configuration

Reply:
That's a fair point... I'll see how much work it'll be to make it optional.